packages/lib/isOutOfBounds.tsx

```diff
@@ -263,6 +269,14 @@ export function isTimeViolatingFutureLimit({
       startOfRangeStartDayInEventTz: periodLimits.startOfRangeStartDayInEventTz.format(),
       endOfRangeEndDayInEventTz: periodLimits.endOfRangeEndDayInEventTz.format(),
     });
+    if (isBeforeRangeStart || isAfterRangeEnd)
+      log.warn("Booking is out of bounds due to range start and end.", {
+        formattedDate: dateInSystemTz.format(),
+        isBeforeRangeStart,
+        isAfterRangeEnd,
+        startOfRangeStartDayInEventTz: periodLimits.startOfRangeStartDayInEventTz.format(),
+        endOfRangeEndDayInEventTz: periodLimits.endOfRangeEndDayInEventTz.format(),
+      });
```

Comment:
Add detailed logs when booking fails due to attempted booking outside of period Range